api/pkg/s3/service.go

```diff
@@ -15,17 +15,14 @@
 package s3
 
 import (
-	"math"
-
-	"github.com/emicklei/go-restful"
-	"github.com/micro/go-micro/client"
-
-	//	"github.com/micro/go-micro/errors"
 	"context"
 	"io"
 	"io/ioutil"
+	"math"
```

Comment:
why do we need "math"?

Reply:
math is not added by this PR, I think I just format the file, and it is just reorganized.